### ISSUE TITLE
CMakeList - Warning

### ISSUE THREAD

**Issue by kiritigowda** [GPUOpen-ProfessionalCompute-Libraries/rpp#63]
Fix required

```
-- Found OPENCL: /usr/lib/x86_64-linux-gnu/libOpenCL.so  
CMake Warning (dev) at /opt/conda/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:426 (message):
  The package name passed to `find_package_handle_standard_args` (OPENCL)
  does not match the name of the calling package (OpenCL).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindOpenCL.cmake:52 (find_package_handle_standard_args)
  CMakeLists.txt:189 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- ROCm OpenCL Found - Set CL_TARGET_OPENCL_VERSION=220
Add Custome Command
OPenCL incude directories are /opt/rocm/opencl/include;/root/rpp/src/include/cl/
```

**Comment by kiritigowda** [comment]
@r-abishek this issue is still present in Ubuntu 18.04 You can look at the Jenkin logs - http://math-ci.rocm.amd.com/blue/organizations/jenkins/compute-rocm-dkms-no-npi-hipclang%2FShort-GPUOpen%2Frpp/detail/master/112/pipeline/249

You need to be on a VPN to view the portal.

```
+ cmake -DBACKEND=OCL -DCMAKE_BUILD_TYPE=Release ../..
-- The C compiler identification is GNU 7.5.0
-- The CXX compiler identification is GNU 7.5.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Radeon Performance Primitives (RPP) Version -- 0.92
-- RPP install path set to -- /opt/rocm/rpp
-- CMake minimum version required: 3.5
-- CMake version found: 3.17.5
CMake Warning (dev) at /opt/cmake-3.17.5/share/cmake-3.17/Modules/FindPackageHandleStandardArgs.cmake:272 (message):
  The package name passed to `find_package_handle_standard_args` (OPENCL)
  does not match the name of the calling package (OpenCL).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindOpenCL.cmake:52 (find_package_handle_standard_args)
  CMakeLists.txt:112 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found OPENCL: /opt/rocm/opencl/lib/libOpenCL.so  
-- ROCm OpenCL Found - Set CL_TARGET_OPENCL_VERSION=220
-- Added custom command for rpp_kernels
-- OpenCL kernels added!
-- OpenCL include directories -- /opt/rocm/opencl/include;/var/jenkins_home/workspace/ipclang_Short-GPUOpen_rpp_master/0yEuU0vtN/rpp/src/include/cl/;/opt/rocm/include
-- amd_rpp set to build with OpenCL backend
-- amd_rpp set to build with ENABLE_SIMD_INTRINSICS
-- Found Boost: /usr/include (found version "1.65.1") found components: filesystem system 
-- Found OpenMP_C: -fopenmp (found version "4.5") 
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP: TRUE (found version "4.5")  
-- Configuring done
-- Generating done
-- Build files have been written to: /var/jenkins_home/workspace/ipclang_Short-GPUOpen_rpp_master/0yEuU0vtN/rpp/build/release
```

**Comment by kiritigowda** [comment]
Fixed with 0.96
